templates/opencraft-nginx.j2

```diff
@@ -0,0 +1,57 @@
+server {
+    listen *:80;
+
+    server_name {{ OPENCRAFT_DOMAIN_NAME }} www.{{ OPENCRAFT_DOMAIN_NAME }};
```

Comment:
Since we set ALLOWED_HOSTS to only {{ OPENCRAFT_DOMAIN_NAME }}, we already have the same behaviour now, with the 404 served by Django instead of nginx.  If you prefer the 404 to come from nginx, I can change it anyway, but I personally don't care enough to change it.